DEMO.md

```diff
@@ -74,12 +100,20 @@ hdx-toolkit update --organization=healthsites --dataset_filter=somalia-healthsit
 ```
 The `from_path` argument indicates the update is to be done from a file, `undo` indicates that the value to be updated is in the `old_value` column. The `from_path` operation can be done without the `undo` flag in which case the file indicated needs to contain, at least, a `dataset_name` column, a `key` column and a `new_value` column.
 
-The `list` command can output multiple comma separated keys to a table, and also to a CSV file specified using the `--output_path` keyword.
+The `list` command can output multiple comma separated keys to a table, and also to a CSV file specified using the `--output_path` keyword. 
 
 ```shell
 hdx-toolkit list --organization=international-organization-for-migration --key=data_update_frequency,dataset_date --output_path=2024-02-05-iom-dtm.csv
 ```
 
+`list` can also output the value of nested keys such as `organization.name` or lists of values such as `tags.name` or `groups.name`. If the `--with_extras` flag is applied then keys within `resources`, `resource_views` and `showcases` can also be seen. The `--with_extras` flag forces multiple queries to HDX per dataset and can be slow, therefore it should only be used if necessary and only for small numbers of datasets at a time. An example of this is as follows:
+
+```shell
+hdx-toolkit list --organization=healthsites --dataset_filter=gibraltar-healthsites --hdx_site=stage --key=resources.name --value=True --with_extras

```

Comment:
It may be easy to forget to pass "with_extras". What if instead you just check if the key has a "resource." or similar?

Reply:
That's a good idea!